acceptance/api_v1_applications_test.go

```diff
@@ -524,4 +527,76 @@ var _ = Describe("Apps API Application Endpoints", func() {
 			})
 		})
 	})
+
+	Context("Logs", func() {
+		Describe("GET api/v1/orgs/:orgs/applications/:app/logs", func() {
+			logLength := 0
+			var (
+				route string
+				app   string
+			)
+
+			BeforeEach(func() {
```

Comment:
From the content of the function I am wondering if that should be an `It` instead of a `BeforeEach` ?!
If it is the latter, maybe hiding the complexity in a utility function ?

Reply:
Done here: https://github.com/epinio/epinio/commit/30bc46fcf0e4734d83a0a47fb91c62266046c652